bin/verify-celoscan.ts

```diff
@@ -0,0 +1,270 @@
+/*******************************************************************
```

Comment:
This code is adapted from https://github.com/k1rill-fedoseev/sourcify-to-etherscan
And fixes some bugs there